### PR TITLE
added PlaceSortingGroup to PlaceParamStructure

### PR DESCRIPTION
**Pull request by herlitze**

### CODE REVIEW

File: OJP/OJP_Places.xsd [hunk 1]
```diff
@@ -201,6 +202,25 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:group name="PlaceSortingGroup">
+		<xs:sequence>
+			<xs:element name="SortingMethod" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>If there are multiple sorting methods supported by the system, the client can choose one of the existing.</xs:documentation>
```
Comment (herlitze) [inline OJP/OJP_Places.xsd:209]:
```suggestion
					<xs:documentation>If there are multiple sorting methods supported by the system, the client can choose one of the existing. Which methods are available needs to be known to the client.</xs:documentation>
```

**Review by ue71603 — DISMISSED** [review]

File: OJP/OJP_Places.xsd [hunk 2]
```diff
@@ -201,6 +202,25 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:group name="PlaceSortingGroup">
+		<xs:sequence>
+			<xs:element name="SortingMethod" type="xs:string" minOccurs="0">
```
[the remark this answers is not in the document]
Comment (sgrossberndt) [reply OJP/OJP_Places.xsd:212]: "Which methods are available needs to be known to the client."

How does the client know? The comment should be improved.

**Review by sgrossberndt — DISMISSED** [review]

File: OJP/OJP_Places.xsd [hunk 3]
```diff
@@ -206,6 +207,25 @@
...
+				<xs:annotation>
+					<xs:documentation>If there are multiple sorting methods supported by the system, the client can choose one of the existing. Which methods are available needs to be known to the client.</xs:documentation>
```
Comment (ue71603) [inline OJP/OJP_Places.xsd:214]:
```suggestion
					<xs:documentation>If there are multiple sorting methods supported by the system, the client can choose one of the existing. Which methods are available needs to be known to the client and may differ from system to system. Therefore use with care.</xs:documentation>
```

File: OJP/OJP_Places.xsd (hunk 2 above)
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP/OJP_Places.xsd:212]: created  update

[the remark this answers is not in the document]
Comment (ue71603) [reply OJP/OJP_Places.xsd:212]: done

[the remark this answers is not in the document]
Comment (trurlurl) [reply OJP/OJP_Places.xsd:212]: Should there be an error / problem code in case of an unsupported sorting method?

**Review by AndreasAtSBB — DISMISSED** [review]
Seems OK, also checkd the HTML text.

File: OJP/OJP_Places.xsd (hunk 2 above)
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP/OJP_Places.xsd:212]: added the problem

**Review by AndreasAtSBB — APPROVED** [review]

**Comment by AndreasAtSBB** [comment]
Looks fine too me.

**Review by trurlurl — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Pull request merged**